Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM ubuntu:14.04
+
+MAINTAINER Doody Parizada doody.parizada @ gmail.com
+
+# install apt dependencies
+RUN apt-get update -y && \
+    apt-get install -y build-essential git python python-dev python-setuptools python-pip && \
+    apt-get build-dep -y python-imaging && \
+    apt-get build-dep -y python-lxml && \
```

Comment:
- why not `apt-get install -y python-imaging python-lxml`?
- maybe it's better to install these via `pip` instead, though you'll need to instead `zlib1g` and some other obscurities